bowling.test.js

```diff
@@ -0,0 +1,56 @@
+const Bowling = require('./bowling.js');
+
+let bowling;
+
+function multipleRolls(rolls, pins) { // function to roll several times
+  for (let i = 0; i < rolls; i++) {
+    bowling.roll(pins);
+  }
+}
+
+describe("Bowling", () => {
+
+beforeEach(() => {          
+    bowling = new Bowling;
+  })
+
+  it("is an instance of Bowling class", () => {
+    expect(bowling).toBeInstanceOf(Bowling);
+  });
```

Comment:
This test is not useful - it just tests that Javascript works, not the behaviour of your class.